cluster/examples/kubernetes/ceph/upgrade-from-v0.8-create.yaml

```diff
@@ -157,7 +157,7 @@ rules:
   verbs: [ "get", "list", "watch", "create", "update", "delete" ]
 ---
 # Aspects of ceph-mgr that require access to the system namespace
-kind: Role
+kind: ClusterRole
```

Comment:
This yaml was specific for 0.8 upgrade, so we won't want to update this file. When we're getting close to release, we will create a new upgrade-from-v0.9 file where this will be included, but we don't need to worry about it right now.

Reply:
Got it, thanks. I dropped the changes to this file.